backend/utils/singleValueInMemoryCache.ts

```diff
@@ -0,0 +1,54 @@
+import moment from 'moment'
+import log from '../log'
+
+type StoredCachedData = {
+  expiryTime: moment.Moment
+  cachedData: any
+}
+
+let cachedValue: StoredCachedData = null
+
+class CacheHelper {
+  private currentDateTime: moment.Moment = moment()
+
+  private ttlInSeconds: number
+
+  constructor(ttlInSeconds: number) {
+    this.ttlInSeconds = ttlInSeconds
+  }
+
+  isExpired(cachedData: StoredCachedData): boolean {
+    return this.currentDateTime.isSameOrAfter(cachedData.expiryTime)
+  }
+
+  makeValueToCache(dataToCache: any): StoredCachedData {
+    return {
+      expiryTime: this.currentDateTime.add(this.ttlInSeconds, 'seconds'),
+      cachedData: dataToCache,
+    }
+  }
+}
+
+export const cacheFactory = (ttlInSeconds) => {
```

Comment:
Unrelated - can you stick default here instead of at the bottom of the file?

Reply:
Nope - this seems to be the only way to do it. Adding the `default export` at the bottom just appeased the linter - it isn't actually used